gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -90,6 +90,18 @@ export default function NxFormLayoutExample() {
           Blue
         </NxRadio>
       </fieldset>
+      <div className="nx-form-group">
+        <label className="nx-label">
+          <span className="nx-label__text">Select</span>
+          <select className="nx-form-select">
+            <option className="nx-form-select__option">Option 1</option>
```

Comment:
What's the purpose of this class?

Reply:
> What's the purpose of this class?

I was trying something and forgo to remove it: https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/407d9a451b1dd4719f51e388ffe90c8e88bc7c55